src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/eltwise.cpp

```diff
@@ -345,7 +354,9 @@ const std::vector<utils::EltwiseTypes>& eltwiseOpTypesBinDyn() {
 
 const std::vector<CPUSpecificParams>& cpuParams_4D() {
     static const std::vector<CPUSpecificParams> cpuParams_4D = {
+#if !defined(OV_CPU_WITH_SHL)
```

Comment:
As I mentioned in the previous comment, If you implement riscv64-specific `filterCPUInfo`, you don't need to insert `#if !defined(OV_CPU_WITH_SHL)` to disable non-planar layouts validation 😃